conf/site-swarm.conf

```diff
@@ -1,6 +1,7 @@
 server {
         listen                  80 default_server;
         server_name             localhost;
+		server_tokens 			off;
```

Comment:
We are using spaces in this file

```suggestion
        server_tokens           off;
```

Reply:
sorry, it's already fixed
![image](https://user-images.githubusercontent.com/8170372/108353289-bccdcf80-71c6-11eb-9215-8151c06a5e93.png)

Reply:
🚀